components/charts/line-charts/DeveloperHoursLineChart.tsx

```diff
@@ -126,8 +48,64 @@ const legends = [
 ]
 const pointSize = 6
 const useMesh = true
+const yScale = {
+    type: 'linear',
+    min: 0,
+    max: 8
+}
+const colors = ['#3066be', '#da5b38', '#3fc1aa']
 
 const DeveloperHoursLineChart = () => {
+    const {
+        devHours: { developers }
+    } = useSelector((state: any) => state.developerHours)
+    const [dataSource, setDataSource] = useState<any>(null)
+
+    const today = new Date()
+    const chartDateLimit = eachDayOfInterval({
+        start: subDays(today, 6),
+        end: today
+    })
+    const formattedDates = chartDateLimit.map((data: any) => {
```

Comment:
Didnt notice these on the last review. Move `chartDateLimit` and `formattedDates` into the useEffect as well. They are only ever being used within there, so they should only be declared where they are being used